backend/src/test/java/gov/cdc/usds/simplereport/service/TestResultUploadServiceTest.java

```diff
@@ -708,6 +710,63 @@ void uploadService_processCsv_handlesEscapedCommas_inTestingLabValues() {
     assertThat(row.getTestingLabPhoneNumber().getValue()).isEqualTo("205-888-2000");
   }
 
+  @Test
+  @SliceTestConfiguration.WithSimpleReportStandardUser
+  void uploadService_processCsv_handlesDefaultDateTimeZone_withValidOrderingProviderAddress() {
+    // GIVEN
+    InputStream input = loadCsv("testResultUpload/test-results-upload-valid-default-dates.csv");
+    UploadResponse response = buildUploadResponse();
+    when(dataHubMock.uploadCSV(any())).thenReturn(response);
+    when(resultsUploaderCachingServiceMock.getSpecimenTypeNameToSNOMEDMap())
+        .thenReturn(Map.of("nasal swab", "000111222"));
+    when(resultsUploaderCachingServiceMock.getCovidEquipmentModelAndTestPerformedCodeSet())
+        .thenReturn(Set.of(ResultsUploaderCachingService.getKey("ID NOW", "94534-5")));
+    var providerAddress = new StreetAddress("400 Main Street", "", "Hayward", "CA", "94540", null);
+    when(resultsUploaderCachingServiceMock.getZoneIdByAddress(providerAddress))
+        .thenReturn(ZoneId.of("US/Pacific"));
+    when(repoMock.save(any())).thenReturn(mock(TestResultUpload.class));
+
+    // WHEN
+    sut.processResultCSV(input);
+
+    // THEN
+    TestResultRow row = getRowFromUpload(dataHubMock);
+    assertThat(row.getTestResultDate().getValue()).isEqualTo("2021-12-23T12:00-08:00");
+    assertThat(row.getOrderTestDate().getValue()).isEqualTo("2021-12-20T12:00-08:00");
+    assertThat(row.getSpecimenCollectionDate().getValue()).isEqualTo("2021-12-20T12:00-08:00");
+    assertThat(row.getTestingLabSpecimenReceivedDate().getValue())
+        .isEqualTo("2021-12-20T12:00-08:00");
+    assertThat(row.getDateResultReleased().getValue()).isEqualTo("2021-12-23T12:00-08:00");
```

Comment:
would these fail when we enter DST?

Reply:
ooo that's a good question -- let me investigate 👀

Reply:
Changed the tests -- let me know what you think!! Thank you for catching that :)